### DISCUSSION TITLE
Input arrays not the same

### DISCUSSION THREAD

**Discussion by SoppierBrute9** [bonsai-rx/bonsai#1521] [Q&A] [answered]
How do I fix the runtime error of "input arrays must have the same shape except in the dimension corresponding to axis"
I am trying to create a workflow in which two cameras are triggered to record synchronously, I am using the tutorial as a guide.

**Comment by bruno-f-cruz** [comment, marked as answer]
You need to make the axis along which you want to concatenate the same. How you do it, its really up to you. You can set the size in the camera acquisition software, or you can use `Crop`, `Resize`, `ResizeCanvas`, etc... They should also have the same number of color channels.
